Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.opensuse.org/devel/bci/sle-15-sp3/images/bci/nodejs
```

Comment:
Surely this should be registry.suse.com/bci/nodejs ? the above container is not tested nor release managed